sci-electronics/kicad/files/kicad-boost-1.61.patch

```diff
@@ -0,0 +1,347 @@
+--- a/common/tool/tool_manager.cpp
```

Comment:
No repoman did not complain about it. This is kind of upstreamed, provided as a workaround for boost issues, but not in any of the repositories as of date. I would like to keep this patch in portage until this changes.